utils-metrics/pom.xml

```diff
@@ -120,6 +120,7 @@
     <dependency>
       <groupId>org.bdgenomics.utils</groupId>
       <artifactId>utils-misc_2.10</artifactId>
+      <scope>compile</scope>
```

Comment:
That being said, if it is just IDEA givin' you the business, I'm inclined to not change it.